diff --git a/python/dist/system_python.bzl b/python/dist/system_python.bzl
index 9367dd0..96b3699 100644
--- a/python/dist/system_python.bzl
+++ b/python/dist/system_python.bzl
@@ -156,12 +156,12 @@ def register_system_python():
 
 def _get_python_version(repository_ctx):
     py_program = "import sys; print(str(sys.version_info.major) + '.' + str(sys.version_info.minor) + '.' + str(sys.version_info.micro))"
-    result = repository_ctx.execute(["python3", "-c", py_program])
+    result = repository_ctx.execute(["python", "-c", py_program])
     return (result.stdout).strip().split(".")
 
 def _get_python_path(repository_ctx):
     py_program = "import sysconfig; print(sysconfig.get_config_var('%s'), end='')"
-    result = repository_ctx.execute(["python3", "-c", py_program % ("INCLUDEPY")])
+    result = repository_ctx.execute(["python", "-c", py_program % ("INCLUDEPY")])
     if result.return_code != 0:
         return None
     return result.stdout
